reference/websockets/sails.io.js/socket.request.md

```diff
@@ -23,10 +46,10 @@ io.socket.request(options, function (data, jwr)){
 ### Example
 
 ```javascript
-io.socket.request({
+io.socket._request({
   method: 'get',
   url: '/user/3/friends',
-  params: {},
+  data: {},
```

Comment:
first argument options needs property `params` not `data` https://github.com/balderdashy/sails-generate-sails.io.js/blob/master/templates/sails.io.js#L871